tests/example_pipelines/compiled_pipeline/example_1/kubeflow_pipeline.yml

```diff
@@ -1,13 +1,10 @@
+# PIPELINE DEFINITION
+# Name: testpipeline
+# Description: description of the test pipeline
```

Comment:
Why is this commented at the top of each pipeline.yml?

Reply:
its generated by kfp